### PR TITLE
Network names and services links update.

### LINKED ISSUE #916: Update network names and services links.
Update all the `Berkeley` testnet mentioning with the `Devnet` and also update all the `minaexplorer` links with the `minascan` (utilizing `Devenet` links instead of `Berkeley`).

### LINKED ISSUE #930: Fix enumerated lists that include code snippets.
image

### PR DESCRIPTION
**Pull request by shimkiv**
Closes #916 
Closes #930

---

Services links provided by `Minascan` were discussed with `Staketab` representative in Discord (`@duccaofficial`).

### CODE REVIEW

File: examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-import './reactCOIServiceWorker';
-import ZkappWorkerClient from './zkappWorkerClient';
-import { PublicKey, Field } from 'o1js';
 import GradientBG from '../components/GradientBG.js';
 import styles from '../styles/Home.module.css';
+import './reactCOIServiceWorker';
+import ZkappWorkerClient from './zkappWorkerClient';
 
 let transactionFee = 0.1;
 const ZKAPP_ADDRESS = 'B62qknCphH8ikE9Bu46dFgamUeYRcE9FtKr5ahh5tcJDf7ibRXJdzWV';
```
Comment (ymekuria) [inline examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx:9]: This can be for another PR, but we should deploy this contract to `devnet`

**Review by ymekuria — APPROVED** [review]
Thanks for updating all of these @shimkiv! The minascan devnet urls are functioning now correct? ie
```
 https://api.minascan.io/node/devnet/v1/graphql
 https://minascan.io/devnet/tx 
```

If so, it would be great if we could move all the updates with these urls to another PR and merge them before HF.

File: examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx (hunk 1 above)
Comment (shimkiv) [reply examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx:9]: I'll deploy one, thanks.

**Comment by shimkiv** [comment]
> If so, it would be great if we could move all the updates with these urls to another PR and merge them before HF.

Actually let me update network names and we can merge it right away without a need to wait for the HF.

File: examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx (hunk 1 above)
Comment (shimkiv) [reply examples/zkapps/04-zkapp-browser-ui/ui/src/pages/index.page.tsx:9]: `Add` contract deployed on `Devnet` at: `B62qpXPvmKDf4SaFJynPsT6DyvuxMS9H1pT4TGonDT26m599m7dS9gP`.

**Review by MartinMinkov — APPROVED** [review]

**Review by ymekuria — APPROVED** [review]
